canonical-web-and-design/jaas-dashboard
# Add model count summary header

## Done

Added model count summary header to the model listings page.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The model count in the top should update as your model data comes in. 


## Details

The data used to generate this count and the data used to sort the table are currently different so you'll likely see different sorting. The immediate follow-up to this branch will be to bring those two in line and expand the logic used to determine how models are categorized. 

## Screenshots

<img width="281" alt="Screen Shot 2019-10-23 at 10 33 58 AM" src="https://user-images.githubusercontent.com/532033/67414613-a4bd5000-f580-11e9-8f54-c058c2542109.png">
